www/src/pages/blog/index.js

```diff
@@ -105,7 +105,7 @@ class BlogPostsIndex extends React.Component {
 export default BlogPostsIndex
 
 export const pageQuery = graphql`
-  query {
+  query BlogPostsIndex {
```

Comment:
Did you mean to add these all back? @pieh added a PR recently to make these optional.

Reply:
Ah dang! Yeah I meant to add these back…`yarn.lock` again I guess. Should I revert that commit?

Reply:
yes please :-)